src/app/(layout-with-navigation)/(simulation)/fin/_components/CongratulationsText.tsx

```diff
@@ -0,0 +1,25 @@
+import Trans from '@/components/translation/Trans'
+import Separator from '@/design-system/layout/Separator'
+import Emoji from '@/design-system/utils/Emoji'
+
+export default function CongratulationsText() {
+  return (
+    <>
+      <h1 className="mt-4">
+        <Trans>
+          <span className="flex gap-2">
+            Vous avez terminé le test ! <Emoji>👏</Emoji>
```

Comment:
Espace insécable avant le !

Reply:
(et avant l'émoji aussi ceci dit)

Reply:
Il serait préférable de seulement traduire le texte d'ailleurs : 

```suggestion
            {t('Vous avez terminé le test\u202f!')}{' '}<Emoji>👏</Emoji>
```

Reply:
On pourrait pas ajouter un plugin prettier ? La flemme d'assumer cette charge mentale 💁‍♂️